### PR TITLE
Modify max code size from 0x6000 to 0x40000

### PR DESCRIPTION
**Pull request by JimmyShi22**

### CODE REVIEW

File: libinitializer/GlobalConfigureInitializer.cpp [hunk 1]
```diff
@@ -58,6 +60,16 @@ bool dev::initializer::getVersionNumber(const string& _version, uint32_t& _versi
     return true;
 }
 
+uint32_t dev::initializer::getVersionNumber(const string& _version)
```
Comment (bxq2011hust) [inline libinitializer/GlobalConfigureInitializer.cpp]: I think this is unnecessary.

File: libinitializer/GlobalConfigureInitializer.cpp [hunk 2]
```diff
@@ -78,6 +90,11 @@ void dev::initializer::initGlobalConfig(const boost::property_tree::ptree& _pt)
     else if (getVersionNumber(version, versionNumber))
     {
         g_BCOSConfig.setSupportedVersion(version, static_cast<VERSION>(versionNumber));
+
+        if (versionNumber < getVersionNumber("2.1.0"))
+        {
+            g_BCOSConfig.setEVMSchedule(dev::eth::FiscoBcosSchedule);
```
Comment (bxq2011hust) [inline libinitializer/GlobalConfigureInitializer.cpp]: this should after parsing versionNumber, and set accord g_BCOSConfig.supportedVersion() or g_BCOSConfig.version()

File: libinitializer/GlobalConfigureInitializer.cpp (hunk 1 above)
Comment (JimmyShi22) [reply libinitializer/GlobalConfigureInitializer.cpp]: It's ok

File: libinitializer/GlobalConfigureInitializer.cpp (hunk 2 above)
Comment (JimmyShi22) [reply libinitializer/GlobalConfigureInitializer.cpp]: done

File: libinitializer/GlobalConfigureInitializer.cpp [hunk 3]
```diff
@@ -78,6 +90,11 @@ void dev::initializer::initGlobalConfig(const boost::property_tree::ptree& _pt)
     else if (getVersionNumber(version, versionNumber))
     {
         g_BCOSConfig.setSupportedVersion(version, static_cast<VERSION>(versionNumber));
+
+        if (g_BCOSConfig.version() < getVersionNumber("2.1.0"))
```
Comment (bxq2011hust) [inline libinitializer/GlobalConfigureInitializer.cpp]:
```suggestion
        if (g_BCOSConfig.version() <= 0x02000000)
```

Comment (bxq2011hust) [reply libinitializer/GlobalConfigureInitializer.cpp]: and please move this if out of the logic of parsing version Number

File: libinitializer/GlobalConfigureInitializer.cpp (hunk 1 above)
Comment (bxq2011hust) [reply libinitializer/GlobalConfigureInitializer.cpp]: I recommend delete it.

**Pull request merged**